requirements.txt

```diff
@@ -1,3 +1,5 @@
+requests
```

Comment:
`requests` is a separate library from `flask`.  In your code, you have `from flask import Flask, request`, which imports the `request` object from the `flask` library.  (Note the singular `request` as opposed to the plural `requests`.

`requests` will become a dependency in our project, but not here.  Please remove.

Reply:
As there are two teams working on Flask related tasks. Is there a specific team that should add flask to the requirements.txt? For example if the get job PR is going to be merged before this post endpoint PR should the get job PR have flask added into their requirements file?